test/models/report_test.rb

```diff
@@ -3,7 +3,19 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'editable?' do
+    user =  users(:alice)
+    user2 = users(:bob)
```

Comment:
```suggestion
    alice =  users(:alice)
    bob = users(:bob)
```

連番は脳に優しくないので避けた方がベターです

Reply:
すいません😭何度も言われていることなのに連番にしてしまいました😭修正しました！